src/endpoints/createEventsEndpoint.js

```diff
@@ -50,7 +51,14 @@ export default (baseUrl: string): Endpoint<ParamsType, Array<EventModel>> => new
         excerpt: decodeHTML(event.excerpt),
         availableLanguages: mapAvailableLanguages(event.available_languages),
         lastUpdate: moment.tz(event.modified_gmt, 'GMT'),
-        hash: event.hash
+        hash: event.hash,
+        featuredImage: event.featured_image ? new FeaturedImageModel({
+          description: event.featured_image.description,
+          thumbnail: event.featured_image.thumbnail[0],
+          medium: event.featured_image.medium[0],
+          large: event.featured_image.large[0],
+          full: event.featured_image.full[0]
+        }) : null
       })
```

Comment:
Do we only have a featured image for events?

Reply:
I think so, at least pages does not have a featured image yet. Maybe the pois, but not sure.

Reply:
I checked it, only events right now.